website/docs/components/button/partials/code/how-to-use.md

```diff
@@ -126,6 +126,25 @@ If the route is external to your current engine, you have to pass `@isRouteExter
 <Hds::Button @text="Back to homepage" @icon="arrow-left" @route="index" />
 ```
 
+### Loading state
+
+If the button needs to toggle between an "idle" and a "loading" state, we suggest to apply a width to it (via inline style or via CSS class) to prevent the button to resize on click (and potentially cause layout shifts):
```

Comment:
```suggestion
If the button needs to toggle between an "idle" and a "loading" state, we suggest applying a width to it (via inline style or CSS class) to prevent the button from resizing on click (and potentially causing layout shifts):
```